lib/composure.bash

```diff
@@ -10,7 +10,7 @@
 
 composure_keywords ()
 {
-    echo "about author example group param version"
+    echo "about author example group param reload version"
```

Comment:
I don't think that you need to add this here. Please remove the reference to `reload` here.